layouts/partials/cookie_consent.html

```diff
@@ -0,0 +1,5 @@
+<script type="text/javascript">
+    window.cookieconsent_options = {{ dict "message" "This website uses cookies." "dismiss" "Got it!" "learnMore" "More info" "link" (.Site.Params.cookie_consent_info_url | absURL)  "theme" "dark-top" | jsonify | safeJS }};
+</script>
+
+<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/cookieconsent2/1.0.10/cookieconsent.min.js"></script>
```

Comment:
Ok, Cookie Consent 3 is like a completely different project and the sample code on their documentation page doesn't work! Ugh. So anyways, I figured it out just now.

![screen shot 2016-11-12 at 18 05 43-fs8](https://cloud.githubusercontent.com/assets/191754/20239164/c9533baa-a902-11e6-94fc-08d86f594f03.png)

About the design, I dunno, there are too many options for color, placement, link text, etc, and I guess it should probably be user configurable (though I think we should make opinionated defaults, and try not to make configuration variables for *every* bell and whistle!).